Example Apps/Example ObjC/VehicleDataManager.m

```diff
@@ -111,8 +111,8 @@ - (void)vehicleDataNotification:(SDLRPCNotificationNotification *)notification {
         return;
     }
 
-    SDLOnVehicleData *onVehicleData = (SDLOnVehicleData *)notification.notification;
-    self.vehicleOdometerData = [NSString stringWithFormat:@"%@: %@ kph", VehicleDataOdometerName, onVehicleData.odometer];
+    SDLOnVehicleData *onVehicleData = (SDLOnVehicleData *)notification;
```

Comment:
Shouldn't this be `(SDLOnVehicleData *)notification.notification;`?

Reply:
https://media.giphy.com/media/xT9IgrStO9Vdk3odgY/giphy.gif